README.md

```diff
@@ -99,36 +78,48 @@ Deploy theme to production via NPM scripts.
 
 ## API
 
-### `command(options, callback)`
+### `command(command[, flags][, options)`
```

Comment:
My decision to leave it the same was just a way to keep this PR simpler. I do agree that having each command available individually would be clearer for the API, though.

The original `command` function was written this way so it was just a wrapper for the arguments passed into `child_process.spawn`, but there's not really a need for that anymore since there's a bit more abstraction involved already.